roles/agnosticv/tasks/generate_catalog_item_tasks.yml

```diff
@@ -9,6 +9,25 @@
     template_namespace: >-
       {{ merged_vars.__meta__.catalog.namespace | default('openshift') }}
 
+- name: Get k8s fact for the catalog item
+  k8s_facts:
```

Comment:
`k8s_facts` is deprecated. Should use `k8s_info`. Likewise in other places where we are using `k8s_facts`.